docs/user-guide/cli-configure-scs-on-headless-linux-os.md

```diff
@@ -103,15 +160,22 @@ The following steps describe how to install and build the credential store binar
     subscription-manager repos --enable rhel-7-for-system-z-optional-source-rpms
     subscription-manager repos --enable rhel-7-for-system-z-optional-debug-rpms
     ```
-6. Install the toolset:
+6. Install the toolset by issuing the following command:
     ```
     yum install devtoolset-11
     ```
-7. After you install the toolset on RHEL V7.x, you can install Zowe CLI.
+7. Install Zowe CLI.
 
-    **Important:** The SCS is installed every time that you install or update Zowe CLI. On RHEL V7.x, ensure that the Red Hat Developer Toolset is enabled every time you install or update Zowe CLI. When you do not enable the toolset, secure credential management is not available on the system. To ensure that the toolset is enabled when you install Zowe CLI, issue the following commands instead of the standard NPM install commands. For example:
+8. [Unlock the keyring manually](#unlocking-the-keyring-manually) or [unlock the keyring automatically](#unlocking-the-keyring-automatically)to load and store credentials **[correct?]**.
+
+    :::info important
+    
+    The secure credential storage capability is installed every time that you install or update Zowe CLI. On RHEL V7.x, ensure that the Red Hat Developer Toolset is enabled every time you install or update Zowe CLI. When you do not enable the toolset, secure credential management is not available on the system. To ensure that the toolset is enabled when you install Zowe CLI, issue the following commands instead of the standard `npm install` commands. **[are these the exact commands, or are these example commands?]**
```

Comment:
@adam-wolfe @zFernand0 - please respond

Reply:
Had to dig around for a bit, but the answer is yes. Since the `@zowe/cli` has prebuilt binaries, people have to enable the dev tool set. And `devtoolset-11` is only for RHEL v7 😋 

Some resources:
- [RHEL Docs: Using other binaries](https://access.redhat.com/documentation/en-us/red_hat_developer_toolset/11/html-single/user_guide/index#sect-binutils-Other)
- [RHEL Docs: List of packages in the devtoolset-11 (with their versions included)](https://access.redhat.com/documentation/en-us/red_hat_developer_toolset/11/html/11.0_release_notes/dts11.0_release#Changes_in_DTS)